<?xml version="1.0"?>

<launch>
   <arg name="robot_name" default="robot"/>
   <arg name="sensor_range" default="1.5"/>
   <arg name="sensor_update_rate" default="10"/>

   <param name="robot_name" type="str" value="$(arg robot_name)"/>
   <rosparam param="victims_models">[victim1,victim2,victim3,victim4]</rosparam>
   <param name="vs_sensor_range" type="double" value="$(arg sensor_range)"/>
   <param name="vs_sensor_update_rate" type="double" value="$(arg sensor_update_rate)"/>
   
   <node name="victim_recognition_system" pkg="payloads" type="victim_recognition_system.py" output="log"/>
    
</launch>	
